src/core/components/otter/Editor.jsx

```diff
@@ -136,6 +142,24 @@ function ctx_reducer(state, op) {
   return {data: state.data, blocks: state.blocks}
 }
 
+function Blocks ({data, block_numbers}) {
+  const first_render = useOnFirstRender()
+
```

Comment:
Sorry if it's obvious, but what's this doing?

Reply:
I'm on mobile atm and my memory is a little fuzzy.

But I think it was to do animations not firing when block builder was empty, or when it became empty, or just first render.

Maybe try removing it and then see if anims are broken when zero blocks and adding, or removing last block.

Can look later on this eve on lappy otherwise

Reply:
Cool. Couldn't find any change when removing it so taken out for now.

Reply:
Fixed possibly-related issue here: https://github.com/keepthatworktoyourself/otter/commit/2c37fe46344595ccac213f8d805b0ef43e34b36d (Only saw an issue with this after integrating into wombat)